src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -80,11 +80,7 @@ const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ fagsak }) =
             <div className={'oppsummering__navigering'}>
                 <Knapp
                     type={'hoved'}
-                    onClick={() => {
-                        aktivBehandling?.type === Behandlingstype.REVURDERING
-                            ? history.push(`/fagsak/${fagsak.id}/vilkår`)
-                            : history.push(`/fagsak/${fagsak.id}/beregning`);
-                    }}
+                    onClick={() => history.push(`/fagsak/${fagsak.id}/oppsummeringberegning`)}
```

Comment:
Burde vi ha et litt bedre navn her? 
```suggestion
                    onClick={() => history.push(`/fagsak/${fagsak.id}/tilkjent-ytelse`)}
```